waspc/examples/todoApp/src/client/pages/Task.tsx

```diff
@@ -6,21 +6,22 @@ import { useAction } from '@wasp/actions'
 import updateTaskIsDone from '@wasp/actions/updateTaskIsDone'
 import getTask from '@wasp/queries/getTask.js'
 import getTasks from '@wasp/queries/getTasks.js'
+import { Task } from '@wasp/entities'
 
-const Todo = (props) => {
+const Todo = (props: any) => {
   const taskId = parseInt(props.match.params.id)
-  const { data: task, isFetching, error } = useQuery(getTask, { id: taskId })
+  const { data: task, isFetching, error } = useQuery<unknown, Task, Task | Task[]>(getTask, { id: taskId })
```

Comment:
The last one is a mistake (I think), don't know what happend there. I'll fix it.

Because we don't have TS docs, the users have to go to definition of `useQuery` to understand to see how to use the types.

Reply:
Ok, feel free to resolve this issue when done.